skietbaan_fe/src/components/Posts.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux';
+import { fetchPosts } from '../actions/postActions'
+
+class Posts extends Component {
+
+  // a constructor that maintanes the state of the component
+ /**  constructor(props ){
+    super(props);
+    this.state = {
+      posts: []
+    }
+  } */
+componentWillMount(){
+  this.props.fetchPosts();
+
+}
+
+componentWillReceiveProps(nextProps){
+  if(nextProps.newPost){
+      this.props.posts.unshift(nextProps.newPost);
+  }
+}
+
+//the method to render the 
+  render() {
+
+    const postItems = this.props.posts.map(post => (
+        <div key={post.id}>
+          <h3>{post.title}</h3>
+          <p>{post.body}</p>
+        </div>
+    ));
+
+    return (
+      <div>
+        {/*<h1>This is the Posts Page</h1> */}
```

Comment:
ghost code

Reply:
Sorted. Kindly review

Reply:
Your code is fine but can be optimized with destructuring of your post object.

Just before your postItems const variable, try this:
var { id: id,
        title: title,
        body: body } = post

Now you can assign the id, title and body without being prefixed with the object name every time.